pallets/aleph/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+#![cfg(test)]
```

Comment:
why do we drop migration tests?

Reply:
It would check nothing as `StorageVersion!=default`.